### PR TITLE
Support both options for sending bytes..

### PR DESCRIPTION
**Pull request by crankydillo**
I'm not enthused about this, but hopefully it will make testing easier.  I had the code written so figured we'd leverage it until we make a final decision.  Hopefully, we won't have this 'backup plan' code.

This does include using eliminating the sink from the 'safe approach'.  However, I'm not 100% sure that is the way we should go.  Still debating that too:(

### CODE REVIEW

**Comment by crankydillo** [comment]
Maven is configured to run most tests against both options.

File: server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java [hunk 1]
```diff
@@ -1,156 +1,90 @@
...
+    private static final boolean USE_FLUSHING =
+        Boolean.parseBoolean(System.getProperty("resteasy.server.reactor-netty.use-flushing", "true"));
+
+    /**
+     * This is the {@link Mono} that we return from {@link ReactorNettyJaxrsServer.Handler#handle(HttpServerRequest,
+     * HttpServerResponse)}
+     */
+    protected final Sinks.Empty<Void> completionSink;
+
+    public ChunkOutputStream(Sinks.Empty<Void> completionSink) {
```
Comment (dnagarajan89) [inline server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java:34]: Do we need public access for this constructor? Can this be protected ?

Comment (dnagarajan89) [reply server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java:34]:
```suggestion
    public ChunkOutputStream(final Sinks.Empty<Void> completionSink) {
```

**Review by dnagarajan89 — CHANGES_REQUESTED** [review]

File: server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java (hunk 1 above)
Comment (crankydillo) [reply server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java:34]: Both good suggestions.  I just realized I opened this up against the wrong branch.

Comment (crankydillo) [reply server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java:34]: If you don't mind, would you make these comments against #2

**Pull request closed without merging**